src/app/shared/components/reject-modal-window/reject-modal-window.component.html

```diff
@@ -0,0 +1,27 @@
+<div fxLayout="column" fxLayoutAlign="center center">
+  <h2 mat-dialog-title>{{ modalTitle }}</h2>
+  <p class="confirmation"> {{ modalDescription }} 
+  </p>
+
+  <form [formGroup]="ReasonFormGroup" fxLayout='column' class="reject-form">
```

Comment:
use please double quotes in the template

Reply:
is updated, thanks